src/index.ts

```diff
@@ -1,47 +1,47 @@
-import lynxRegistryJson from '../deployment/artifacts/lynx.json';
-import mainnetRegistryJson from '../deployment/artifacts/mainnet.json';
-import tapirRegistryJson from '../deployment/artifacts/tapir.json';
+import lynxRegistryJson from "../deployment/artifacts/lynx.json";
+import mainnetRegistryJson from "../deployment/artifacts/mainnet.json";
+import tapirRegistryJson from "../deployment/artifacts/tapir.json";
 
-export type Abi = unknown
+export type Abi = unknown;
 
 export interface DeployedContract {
   address: string;
   abi: Abi;
 }
 
 // Only expose contracts that are used in the SDK
-export const contractNames = [
-  'Coordinator',
-  'GlobalAllowList',
-  'SubscriptionManager',
-] as const;
+export const contractNames = ["Coordinator", "GlobalAllowList", "SubscriptionManager"] as const;
 
-export type ContractName = (typeof contractNames)[number]
+export type ContractName = (typeof contractNames)[number];
 
 export interface Contract {
   name: ContractName;
   abi: Abi;
 }
 
-export type ContractRegistry = Record<string, Record<string, DeployedContract>>
+export type ContractRegistry = Record<string, Record<string, DeployedContract>>;
 
 export const domainRegistry: Record<string, ContractRegistry> = {
   lynx: lynxRegistryJson,
   tapir: tapirRegistryJson,
   mainnet: mainnetRegistryJson,
 };
 
-export type Domain = 'mainnet' | 'oryx' | 'tapir' | 'lynx'
+export type Domain = "mainnet" | "oryx" | "tapir" | "lynx";
```

Comment:
@piotr-roslaniec Should we get rid of `oryx` here, or is that still needed for something?

Reply:
Hmm, it's not being used anymore for PRE right?

Reply:
Only Masterfile uses oryx (although I'm not sure how much), but I thought our latest code already deprecated `oryx` anyway i.e. Masterfile must use an old version of `nucypher-ts`/`taco-web` with `oryx`.

(cc @cygnusv , @KPrasch  in case I'm mistaken)